tidb-cloud/tidb-cloud-faq.md

```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
 
 ## Security FAQs
 
-### How does TiDB protect data privacy and ensure security?
+### Is TiDB Cloud secure?
 
-Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.
+In TiDB Cloud, all data at rest is encrypted, and all network traffic is encrypted using Transport Layer Security(TLS). 
 
-### Can TiDB Cloud run in our VPC?
+- Encryption of data at rest is automated using encrypted storage volumes. 
+- Encryption of data in transit between your client and your cluster is automated using TiDB Cloud web server TLS and TiDB cluster TLS.
 
-No. TiDB Cloud runs on the PingCAP VPC, but the data and traffic are encrypted by default. So you do not need to worry about data privacy issues.
+### How does TiDB Cloud encrypt my business data?
+
+TiDB Cloud uses storage volume encryption by default for your business data at rest, including your database data and backup data. TiDB Cloud requires TLS encryption for data in transit, and also requires component-level TLS encryption for data in your database cluster between TiDB, PD, TiKV, and TiFlash.
+
+To get more specific information about business data encryption in TiDB Cloud, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
+
+### What versions of TLS does TiDB Cloud support?
+
+All connections to the cluster over the internet use TLS 1.2 or 1.3.
+
+### Can I run TiDB Cloud in my VPC?
+
+No. TiDB Cloud is Database-as-a-Service (DBaaS) and runs only in the TiDB Cloud VPC. As a cloud computing managed service, TiDB Cloud provides access to a database without requiring the setup of physical hardware and the installation of software.
+
+### Is my TiDB cluster secure?
+
+In TiDB Cloud, you can use either a Dedicated Tier cluster or a Serverless Tier cluster according to your needs.
+
+For Dedicated Tier clusters, TiDB Cloud ensures cluster security with the following measures:
+
+- Creates independent sub-accounts and VPCs for each cluster.
```

Comment:
sub-account or sub-accounts for each cluster?

Reply:
it is better to change sub-account into customer account

Reply:
agree to delete this sentences